apps/arm/main.cpp

```diff
@@ -13,6 +13,13 @@
 #include "CANBuffer.h"
 #include "Servo.h"
 
+// maybe these can be defined more formally in some config file?
```

Comment:
@jetkov should we perhaps have a constants file or folder of files that defines the interaction between ros and hw?

Reply:
Yeah, ideally we would have a set of files that is shared by both projects, that includes the CAN ID assignments as well as the payload structs for mixed-data payloads (protocol buffers).